src/models/poi/index.ts

```diff
@@ -29,3 +32,14 @@ export type Pois = Record<string, PoiData>;
 export type PoiOrNull = Poi | null;
 
 export default Poi;
+
+export interface PoiCategory {
```

Comment:
I think `category` can be changed to `target` to better reflect the data store in it.

Reply:
Also I think we do not need to separate it to another interface export. We may not using it. Writing it directly in `PoiData` is fine.

This suggestion also applies to `PoiStatus`.